README.md

```diff
@@ -291,6 +291,53 @@ spark-shell --jars /path/to/spark-exasol-connector-assembly-*.jar
   parameter. However, it is not advised to limit this value in production
   environment.
 
+## Dependencies
+
+In this section, we define all the dependencies together with their licenses
+that are required for building, testing and running the connector.
+
+The Java 8 is required for compiling and building the project. In the future
+versions of Spark, we are planning to change to the newer JVM versions.
+
+### Runtime Dependencies
+
+| Dependency                                  | Purpose                                                         | License              |
+|---------------------------------------------|-----------------------------------------------------------------|----------------------|
+| [Exasol JDBC][exasol-jdbc-link]             | Accessing Exasol using JDBC and sub-connections                 | MIT License          |
+| [Spark Core][spark]                         | Apache Spark core libraries for optimized computation           | Apache License 2.0   |
+| [Spark SQL][spark-sql-link]                 | Apache Spark higher-level SQL and Dataframe interface libraries | Apache License 2.0   |
+
+### Test Dependencies
+
+| Dependency                                  | Purpose                                                         | License              |
+|---------------------------------------------|-----------------------------------------------------------------|----------------------|
+| [Scalatest][scalatest-link]                 | A testing tool for Scala and Java developers                    | Apache License 2.0   |
+| [Scalatest Plus][scalatestplus-link]        | An integration support between Scalatest and Mockito            | Apache License 2.0   |
+| [Mockito Core][mockitocore-link]            | A mocking framework for unit tests                              | MIT License          |
+| [Testcontainers JDBC][tcont-jdbc-link]      | Testcontainers JDBC to help create JDBC based Docker containers | MIT License          |
```

Comment:
Why not use `exasol-testcontainers`?

Reply:
The real reason is actually Java 11 :)
Until Spark 3.x.x version is released with Java 11, I cannot switch to exasol-containers.
I can easily use from Scala the `exasol-testconatainer` library.

Reply:
Okay, time for an upgrade soon. Sonarcloud already wrote to me that they are discontinuing scans for Java 8 projects.